OpenAPI Specification.json

```diff
@@ -355,7 +355,7 @@
           "distributors": {
             "type": "array",
             "items": {
-              "$ref": "#/components/schemas/DistributorSummary"
```

Comment:
DistributorSummary should be removed if not used after this change

Reply:
Sure, go ahead. Make the change.

Reply:
@ricsv would you complete the PR